source/isaaclab/isaaclab/sensors/camera/tiled_camera.py

```diff
@@ -92,6 +92,19 @@ def __init__(self, cfg: TiledCameraCfg):
             )
         super().__init__(cfg)
 
+        # HACK: we need to disable instancing for semantic_segmentation and instance_segmentation_fast to work
+        if int(get_version()[2]) == 4 and int(get_version()[3]) == 5:  # checks for Isaac Sim v4.5:
+            if "semantic_segmentation" in self.cfg.data_types or "instance_segmentation_fast" in self.cfg.data_types:
+                carb.log_warn(
+                    "Isaac Sim 4.5 introduced a bug in TiledCamera when outputting instance and semantic segmentation"
+                    " outputs for instanceable assets. As a workaround, the instanceable flag on assets will be"
+                    " disabled in the current workflow and may lead to longer load times and increased memory usage."
+                )
+                stage = omni.usd.get_context().get_stage()
```

Comment:
You could use this?
 https://isaac-sim.github.io/IsaacLab/main/source/api/lab/isaaclab.sim.html#isaaclab.sim.utils.make_uninstanceable

Reply:
Nevermind. That is only for a prim.